src/Security/src/Authorization.Certificate/ApplicationInstanceCertificate.cs

```diff
@@ -0,0 +1,62 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics.CodeAnalysis;
+using System.Text.RegularExpressions;
+
+namespace Steeltoe.Security.Authorization.Certificate;
+
+internal sealed class ApplicationInstanceCertificate
+{
+    // This pattern is found on certificates issued by Diego
+    private const string CloudFoundryInstanceCertificateSubjectRegex =
```

Comment:
Please use precompiled static Regex instances (or try the source generator, which is mature in .NET 8)